eos-firewall-localonly.service

```diff
@@ -0,0 +1,12 @@
+# Run eos-firewall-localonly before NetworkManager starts
+
+[Unit]
+Description=Endless local-only firewall
+Before=NetworkManager.service
+
+[Service]
+Type=oneshot
```

Comment:
It’s a bit counterintuitive that `sudo systemctl start eos-firewall-localonly.service` will enable the firewall, but `sudo systemctl stop eos-firewall-localonly.service` will *not* stop it.